packages/dd-trace/src/debugger/devtools_client/snapshot/processor.js

```diff
@@ -0,0 +1,241 @@
+'use strict'
+
+module.exports = {
+  processRawState: processProperties
+}
+
+// Matches classes in source code, no matter how it's written:
+// - Named:                          class MyClass {}
+// - Anonymous:                      class {}
+// - Named, with odd whitespace:     class\n\t MyClass\n{}
+// - Anonymous, with odd whitespace: class\n{}
+const CLASS_REGEX = /^class\s([^{]*)/
+
+function processProperties (props, maxLength) {
+  const result = {}
+
+  for (const prop of props) {
+    // TODO: Hack to avoid periods in keys, as EVP doesn't support that. A better solution can be implemented later
+    result[prop.name.replaceAll('.', '_')] = getPropertyValue(prop, maxLength)
+  }
+
+  return result
+}
+
+function getPropertyValue (prop, maxLength) {
+  // Special case for getters and setters which does not have a value property
+  if ('get' in prop) {
+    const hasGet = prop.get.type !== 'undefined'
+    const hasSet = prop.set.type !== 'undefined'
+    if (hasGet && hasSet) return { type: 'getter/setter' }
+    if (hasGet) return { type: 'getter' }
+    if (hasSet) return { type: 'setter' }
+  }
+
+  switch (prop.value?.type) {
+    case 'object':
+      return getObjectValue(prop.value, maxLength)
+    case 'function':
+      return toFunctionOrClass(prop.value, maxLength)
+    case undefined: // TODO: Add test for when a prop has no value. I think it's if it's defined after the breakpoint?
+    case 'undefined':
+      return { type: 'undefined' }
+    case 'string':
+      return toString(prop.value.value, maxLength)
+    case 'number':
+      return { type: 'number', value: prop.value.description } // use `descripton` to get it as string
+    case 'boolean':
+      return { type: 'boolean', value: prop.value.value === true ? 'true' : 'false' }
+    case 'symbol':
+      return { type: 'symbol', value: prop.value.description }
+    case 'bigint':
+      return { type: 'bigint', value: prop.value.description.slice(0, -1) } // remove trailing `n`
+    default:
+      // As of this writing, the Chrome DevTools Protocol doesn't allow any other types than the ones listed above, but
+      // in the future new ones might be added.
+      return { type: prop.value.type, notCapturedReason: 'unknown' }
```

Comment:
make more informative message `unknown object type ${prop.value.type}`

Reply:
I assume that's because the `type` property is not exposed in the UI?

Reply:
I updated the text slightly and checked how it looks in the UI, and if I for example remove support for strings (just for this test), this is what the UI shows:

<img width="234" alt="image" src="https://github.com/user-attachments/assets/6eb1b93d-a4ef-4116-8c10-98f925533091">

Isn't it enough to have `myString = string` on the line above and this this error message, or do you want to repeat the word `string` in the error message as well?

Reply:
I've updated the PR with the above message for now